src/components/CodeEditor/CodeEditor.js

```diff
@@ -112,7 +118,27 @@ class CodeEditor extends Component {
                   background: colors.darker,
                   color: colors.white,
                 }}>
-                <MetaTitle onDark={true}>Live JSX Editor</MetaTitle>
+                <MetaTitle onDark={true}>
+                  Live JSX Editor
+                  {!showCompiledJsx && (
+                    <a
+                      role="button"
+                      tabIndex="0"
+                      css={{cursor: 'pointer'}}
+                      onClick={this._showCompiledJsxPreview.bind(this)}>
+                      View Compiled JSX
+                    </a>
+                  )}
+                  {showCompiledJsx && (
+                    <a
+                      css={{cursor: 'pointer'}}
+                      role="button"
+                      tabIndex="0"
+                      onClick={this._hideCompiledJsxPreview.bind(this)}>
+                      Back to JSX Editor
+                    </a>
+                  )}
+                </MetaTitle>
```

Comment:
How do you feel about using a checkbox to toggle JSX mode instead of a button? I think it might look better 😄 

![screen shot 2017-10-12 at 10 23 20 am](https://user-images.githubusercontent.com/29597/31509749-69504edc-af37-11e7-9bf3-226650a417c3.png)